addons/mail/static/tests/activity/activity_tests.js

```diff
@@ -0,0 +1,44 @@
+/** @odoo-module **/
+
+import { Activity } from "@mail/activity/activity";
+import { activityData } from "@mail/../tests/activity/activityData";
+import { click, getFixture, mount } from "@web/../tests/helpers/utils";
+import { makeTestEnv, TestServer } from "../helpers/helpers";
+
+let target;
+
+QUnit.module("mail", (hooks) => {
+    hooks.beforeEach(async () => {
+        target = getFixture();
+    });
+
+    QUnit.module("activity");
+
+    QUnit.test("activity", async (assert) => {
+        const server = new TestServer();
+        const env = makeTestEnv((route, params) => {
+            if (route === "/web/dataset/call_kw/mail.activity/unlink") {
+                assert.step("/web/dataset/call_kw/mail.activity/unlink");
+            }
+            return server.rpc(route, params);
+        });
+        await mount(Activity, target, {
+            env,
+            props: { data: activityData },
+        });
+        assert.equal(
+            document.querySelector(".o-mail-activity-name").textContent,
+            activityData.activity_type_id[1]
+        );
```

Comment:
It's not clear the relevant part of asserting this.
If it was in its own named test (and optionally an assertion comment), then would make much more sense.
Also this relies on structure of data that is being passed to the component.
It's alright if the data are well structured. IMO they are not, hence why it's very hard to understand this assertion by itself.

Reply:
This part of the test is kind of stupid anyway, I m not sur we need to test this. I ll just remove it